storage.tf

```diff
@@ -28,6 +28,11 @@ resource "google_storage_bucket" "vault" {
   location      = upper(var.storage_bucket_location)
   storage_class = upper(var.storage_bucket_class)
 
+  logging      {
+    log_bucket        = var.log_bucket_name
+    log_object_prefix = var.log_object_prefix
```

Comment:
this one by default will use the bucket name, atm we dont want to alter the prefix to probably can drop this from the module